### ISSUE TITLE
Crossterm Reset != Black

### ISSUE THREAD

**Issue by joshka** [Canop/coolor#2]
Hi there. I was taking a read of the source and noticed that `crossterm::Color::Reset` gets translated to Black. 

https://github.com/Canop/coolor/blob/7203668228ae4fa44e8dc52b6997fe2416d82609/src/color.rs#L86

This is correct only for terminal themes that have their background color and black set to the same value (e.g. all light themes are excluded by this).

Visually, here's the output of the color example from Ratatui (note the color names are Ratatui names, not crossterm ones):

![colors](https://camo.githubusercontent.com/c23397fe86d2350d2c6a24eca587ee8d0e02231a7fdbd7a46232e41ecf6f10fe/68747470733a2f2f7668732e636861726d2e73682f7668732d325a43715962546258416141536e635565576b74317a2e676966)

I'm not sure what the right design for this should be in `coolor` as it seems like it would be use case dependent, but perhaps adding an explicit `Reset` variant to the color enum. Do you think it would be worth marking this as non-exhaustive to allow adding other color models?

**Comment by Canop** [comment]
The thing is, coolor deals with colors and Reset isn't really a color.
Reset should probably be normally dealt with before calling `into`. Maybe with a function returning an `Option<Color>`.

**Comment by joshka** [comment]
Yeah - that was my sort of thinking too, but I wasn't sure what sorts of use cases would lead to using that method.

My interest in this particular crate was mainly piqued by wondering whether there is value to adding conversions for Ratatui colors.

**Comment by Canop** [comment]
> My interest in this particular crate was mainly piqued by wondering whether there is value to adding conversions for Ratatui colors.

I don't know if that would be useful but it would be clean and straightforward from what I see of ratatui Color, so go ahead if you want.
